packages/webapp/webapp_server.js

```diff
@@ -818,9 +816,16 @@ WebAppInternals.setInlineScriptsAllowed = function (value) {
   WebAppInternals.generateBoilerplate();
 };
 
+// this is the default implementation, can be overridden below
+WebAppInternals.bundledJsCssUrlRewriteHook = function (url) {
```

Comment:
Yeah, I'll back out that change. I was originally thinking of calling it from autoupdate, but no longer.